### DISCUSSION TITLE
Can Live Preview show alerts?

### DISCUSSION THREAD

**Discussion by Meelperd** [phcode-dev/phoenix#1734] [Q&A]
An alert (like onclick=alert) doesn't show in the native live preview, but it does in Chrome. Is there a setting or extension to enable it, or is it only available in browsers?

**Comment by abose** [comment]
Which os is this?

**Reply by Meelperd** [reply]
Win 10

**Reply by abose** [reply]
Only 1 alert is permitted inside the editor per 3 seconds, so as to not hang the editor. But within browser previews its unrestricted.

**Reply by Meelperd** [reply]
Thanks, Arun!
It seems like "alert" is not executing in the editor's live preview. Please see the comparison here:
![Comparison](https://github.com/phcode-dev/phoenix/assets/95029064/5cb3ba0f-9af7-4c82-9fc0-d30333ade01f)

**Reply by abose** [reply]
It appears that it's a problem only in Windows desktop apps, and is linked to an issue with our underlying app shell Tauri: https://discordapp.com/channels/616186924390023171/1258736272814374964

This should anyway be fixed by the next release. Thanks for logging the issue.

**Reply by abose** [reply]
Fixed via: https://github.com/phcode-dev/phoenix/pull/1750
